logstash-filter-verifier.go

```diff
@@ -89,7 +93,10 @@ func runTests(logstashPath string, tests []testcase.TestCase, configPaths []stri
 			} else {
 				message += "\nThe process wrote nothing to its logfile."
 			}
-			return errors.New(message)
+			if err != nil {
+				return errors.New(message)
+			}
+			userError("Logstash output:\n%s", message)
```

Comment:
We can probably just output `message` here. As it currently stands it'll look like this (with a correction for my previous comment):

```
Running tests in client-decoration.json...
Logstash output:
Process output:
Sending logstash logs to /tmp/638530082.

Log:
{:timestamp=>"2017-02-02T20:03:43.426000+0000", :message=>"Reading config file", :config_file=>"/tmp/346208537/50-http-access-filter.conf.j2", :level=>:debug, :file=>"logstash/config/loader.rb", :line=>"69", :method=>"local_config"}
...
```